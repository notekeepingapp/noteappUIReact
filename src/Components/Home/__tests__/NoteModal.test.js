import React from 'react';
import { shallow } from 'enzyme';
import NoteModel from '../NoteModal';


describe('Note Model rendering', () => {
  it('should match NoteModel', () => {
    const note = {
      noteTitle: 'Anju',
      noteContent: 'admin1',
    };
    const component = shallow(<NoteModel note={note} />);
    expect(component).toMatchSnapshot();
  });
});
