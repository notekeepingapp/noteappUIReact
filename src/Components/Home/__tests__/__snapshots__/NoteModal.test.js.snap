// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note Model rendering should match NoteModel 1`] = `ShallowWrapper {}`;
